ci/lib/templates.lib.txt

```diff
@@ -72,43 +72,12 @@ function remote_upload {
 
 (@- end @)
 
-(@ def run_cpp_unit_tests(): -@)
+(@ def ctest_bash_task(path, timeout=300, parallel=6): -@)
 set -ueo pipefail
 (@= remote_functions() @)
-remote_shell cmake -E chdir build/cppcache/test ctest -C ${CMAKE_CONFIG} -j8 --timeout=500 --output-on-failure --schedule-random
```

Comment:
Random side note:  I don't think the unit tests respect the `--schedule-random` flag through cmake, though I'm having trouble finding evidence in docs one way or the other.  I know that executing the binary directly like `apache-geode_unittests --gtest_shuffle` definitely shuffles the unit tests, though that would destroy the simplification here

Reply:
Eh! Random isn't important there unless you're writing bad tests. I errored on the side of consistency in the args.